app.js

```diff
@@ -8,7 +8,15 @@ const {
 const connector = new LCUConnector();
 const { app, dialog } = electron;
 const { BrowserWindow } = electron;
+const DiscordRPC = require('discord-rpc');
+
 const root = __dirname + '/app';
+
+const clientId = '616399117580763176';
```

Comment:
Change this to `616399159322214425`. It's an app I made to be sure there won't be any funny business in the future.

Reply:
changed it to the clientId provided however the assets are not on the discord ap

Reply:
Uploaded them